### ISSUE TITLE
Organize test/mason

### ISSUE THREAD

**Issue by ben-albrecht** [chapel-lang/chapel#15683]
Organize `test/mason` to make it clearer about what commands are being tested.

Directory hierarchy should follow commands that are being tested:

```
mason/
  new/
  init/
  add/
  rm/
  update/
  ...
```

Not high priority, but I wanted to document the desire. Related to https://github.com/chapel-lang/chapel/issues/5599

**Comment by ankingcodes** [comment]
I have been thinking of doing this for a long time. May I work on it ?

**Comment by ben-albrecht** [comment]
Absolutely! Let me know if there are tests you run into that don't clearly fit into this scheme.

**Comment by jabraham17** [comment]
Closing, mason tests are pretty much organized
